packages/webhooks/tasks/sendWebhook.js

```diff
@@ -22,11 +23,11 @@ const logger = getLogger('sendWebhook')
 
 
 async function sendWebhook (subscriptionId) {
+    const taskId = this.id || uuid()
```

Comment:
Do we need to add this to Async Local Storage?

Reply:
I'm not certain yet. We already utilize this technique in other tasks.